docs/source/preparing_datasets/dataset_format.md

```diff
@@ -0,0 +1,22 @@
+# Dataset Format
+
+## Introduction
+
+To use StreamingDataset, one must convert raw data into one of our supported serialized dataset formats. With massive datasets, our serialization format choices are critical to the ultimate observed performance of the system. For deep learning models, we need extremely low latency cold random access of individual samples granularity to ensure that dataloading is not a bottleneck to training.
+
+StreamingDataset is compatible with any data type, including **images**, **text**, **video**, and **multimodal** data. StreamingDataset supports the following formats:
+ * MDS (Mosaic Data Shard, most performant), through {class}`streaming.MDSWriter`
+ * XSV, through {class}`streaming.XSVWriter`
+ * JSONL, through {class}`streaming.JSONWriter`
+These formats can encode and decode most Python objects. For a high-level explanation of how dataset writing works, check out the [main concepts](../getting_started/main_concepts.md#Dataset-conversion) page. The [Dataset Conversion Guide](basic_dataset_conversion_guide.md) shows how to use the {class}`streaming.MDSWriter` to convert your raw data to supported file formats. For large datasets, use the [Parallel Dataset Conversion](parallel_dataset_conversion.ipynb) guide.
```

Comment:
nit: new line before `For a high-level explanation`.

Reply:
done